LauraBeatris/floripamais
# Self-host Inter variable font

## Description 

Self-host Inter variable font. By using a variable font, won't be necessary to request and preload several font files, since all the styles are now placed in a single file.

The font is being cached, that's why the `font-display` descriptor is being set to `optional`

## How to test

- Load the website for the first time and inspect the font request
- Hard refresh the website, inspect the font request again and see that it's now cached
